extensions/interactions/CodeRepl/CodeRepl.py

```diff
@@ -23,8 +23,12 @@ class CodeRepl(base.BaseInteraction):
     name = 'Code Editor'
     description = 'Allows learners to enter code and get it evaluated.'
     display_mode = base.DISPLAY_MODE_SUPPLEMENTAL
-    is_trainable = False
-    _dependency_ids = ['skulpt', 'codemirror']
+    is_trainable = True
```

Comment:
Given that we already have is_terminal, is_linear, etc. -- perhaps is_trainable would be better, to follow the existing pattern.